packages/odyssey-react/src/components/Select/Select/index.tsx

```diff
@@ -176,4 +178,5 @@ Select.OptionGroup = SelectOptionGroup;
 
 Select = withStyles(styles)(Select);
 
+export type { ChoicesHTMLSelectElement };
```

Comment:
rather than exporting this type as part of our public API (it will be funneled all the way up to `import * from "@okta/odyssey-react"` because of how we structure our barrel files) we could move it into a sibling `types.ts` file that is imported directly where needed but not exported from this component module

Reply:
Good solution, I was thinking about that too but didn't have a great idea on how to not change the types API surface - modifying.

Reply:
Just bumping this feedback - I see the type is still being exported thru to our top level API. Should be as simple as removing this line and using an import like this where the type is needed for tests or other logic:

import type { ChoicesSelectHTMLElement } from './types'